docs/atlascli/command/atlas-backups-schedule-update.txt

```diff
@@ -0,0 +1,125 @@
+.. _atlas-backups-schedule-update:
+
+=============================
+atlas backups schedule update
+=============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Update a snapshot backup policies for a cluster.
```

Comment:
```suggestion
Update a snapshot backup policy for a cluster.
```

Reply:
@sarahsimpers You can upgrade multiple policies by using the flag `--policy` many times. Should it stay as **policies**?

Reply:
@Razeer123 Thank you, in that case please just remove the "a" so that it reads "Update snapshot backup policies..."

Reply:
Done!